libheif/0001-Edit-CMakeLists.patch

```diff
@@ -0,0 +1,79 @@
+From 9d98552f1c053e9b3f50ceab09f05a8664333fae Mon Sep 17 00:00:00 2001
+From: L4cache <42721712+L4cache@users.noreply.github.com>
+Date: Sat, 7 Dec 2024 08:54:41 +0800
+Subject: [PATCH] Edit CMakeLists
+
+This patch updates VVdeC version and bypasses some problems during build process and not configurable.
+---
+ CMakeLists.txt        | 27 ++-------------------------
+ heifio/CMakeLists.txt |  2 +-
+ 2 files changed, 3 insertions(+), 26 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index c26ea684..f4c56a81 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -160,7 +160,7 @@ plugin_option(VVDEC "vvdec VVC decoder (experimental)" OFF OFF)
+ if (WITH_VVDEC)
+     # TODO: how to do configure vvdec cleanly?
+     find_package(Threads REQUIRED)
+-    find_package(vvdec 2.3.0)
++    find_package(vvdec 3.0.0)
+     if (vvdec_FOUND)
+         set(vvdec_LIBRARIES vvdec::vvdec)
+     endif()
+@@ -456,7 +456,7 @@ if (WITH_HEADER_COMPRESSION OR WITH_UNCOMPRESSED_CODEC)
+         message("zlib not found")
+     endif()
+ 
+-    find_package(Brotli)
++    # find_package(Brotli)
+     if (Brotli_FOUND)
+         message("Brotli found")
+         list(APPEND REQUIRES_PRIVATE "libbrotlidec")
+@@ -505,29 +505,6 @@ if(WITH_EXAMPLES)
+     add_subdirectory (examples)
+ endif()
+ 
+-# --- API documentation
+-
+-# check if Doxygen is installed
+-find_package(Doxygen)
+-if (DOXYGEN_FOUND)
+-    # set input and output files
+-    set(DOXYGEN_IN ${CMAKE_CURRENT_SOURCE_DIR}/libheif/Doxyfile.in)
+-    set(DOXYGEN_OUT ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile)
+-
+-    # request to configure the file
+-    configure_file(${DOXYGEN_IN} ${DOXYGEN_OUT} @ONLY)
+-    message("Doxygen build started")
+-
+-    # note the option ALL which allows to build the docs together with the application
+-    add_custom_target( doc_doxygen ALL
+-        COMMAND ${DOXYGEN_EXECUTABLE} ${DOXYGEN_OUT}
+-        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+-        COMMENT "Generating API documentation with Doxygen"
+-        VERBATIM )
+-else (DOXYGEN_FOUND)
+-  message("Doxygen tool needs to be installed to generate the API documentation")
+-endif (DOXYGEN_FOUND)
+-
+ # --- Testing
+ 
+ option(ENABLE_COVERAGE "" OFF)
+diff --git a/heifio/CMakeLists.txt b/heifio/CMakeLists.txt
+index 920eeca5..46010dc9 100644
+--- a/heifio/CMakeLists.txt
++++ b/heifio/CMakeLists.txt
+@@ -25,7 +25,7 @@ target_compile_definitions(heifio
+         LIBHEIF_EXPORTS
+         HAVE_VISIBILITY)
+ 
+-find_package(TIFF)
++# find_package(TIFF)
```

Comment:
What's the error with this one? linking issues?

Reply:
Yes, and there's also "built tiff" vs. "pacman tiff" situation.
"built tiff" causes configuration failure, "pacman tiff" has linking issues.